BUILD.md

```diff
@@ -21,20 +21,6 @@ make setup
 
 Run `make help` to get a list of all available targets.
 
-## Adding Dependencies
-
-To add a dependency, use the following command:
-
-```shell
-make vendor-get pkg=<package-uri>
-```
-
-For example, you could add a dependency on `istio.io/istio` like so:
-
-```shell
-make vendor-get pkg=istio.io/istio
-```
-
```

Comment:
Instead of removing this, can you add support for these commands leveraging [Go Modules](https://github.com/golang/go/wiki/Modules#daily-workflow) commands?

Reply:
Done, added a couple of make targets to make `go mod` operations easier